scripts/generate-locales.ts

```diff
@@ -456,23 +531,26 @@ for (const locale of locales) {
     console.error(error);
   }
 
-  const localizedFaker = `faker${locale.replace(/^([a-z]+)/, (part) =>
-    part.toUpperCase()
-  )}`;
+  const localizedFaker = toFakerExportName(locale);
 
   localeIndexImports += `import { faker as ${localizedFaker} } from './${locale}';\n`;
   localeIndexExportsIndividual += `export { faker as ${localizedFaker} } from './${locale}';\n`;
   localeIndexExportsGrouped += `  ${locale}: ${localizedFaker},\n`;
   localesIndexImports += `import ${locale} from './${locale}';\n`;
   localesIndexExportsIndividual += `export { default as ${locale} } from './${locale}';\n`;
   localesIndexExportsGrouped += `  ${locale},\n`;
-  localizationLocales += `| \`${locale}\` | ${localeTitle} | \`${localizedFaker}\` |\n`;
+  const linkedLocale =
+    locale === 'base' ? '`base`' : `[\`${locale}\`](/locales/${locale}.md)`;
+  localizationLocales += `| ${linkedLocale} | ${localeTitle} | \`${localizedFaker}\` |\n`;
 
   promises.push(
     // src/locale/<locale>.ts
     // eslint-disable-next-line unicorn/prefer-top-level-await -- Disabled for performance
     generateLocaleFile(locale),
 
+    // eslint-disable-next-line unicorn/prefer-top-level-await -- Disabled for performance
+    generateLocaleDocumentation(locale),
```

Comment:
For consistency - could you add a comment with the output path over the eslint-disable comment?


```suggestion
    // /docs/locales/*.md
    // eslint-disable-next-line unicorn/prefer-top-level-await -- Disabled for performance
    generateLocaleDocumentation(locale),
```